package.json

```diff
@@ -10,6 +10,7 @@
     "buildstatic": "webpack --config=webpack.static.config.babel.js --progress",
     "build": "npm run buildjs && npm run buildstatic",
     "build:prod": "npm run buildjs:prod && npm run buildstatic",
+    "build:prod:travis": "cross-env NODE_ENV=production webpack -p && webpack --config=webpack.static.config.babel.js",
```

Comment:
Fra CMD doc:
> command1 && command2
Use to run the command following && only if the command preceding the symbol is successful. Cmd.exe runs the first command, and then runs the second command only if the first command completed successfully.